es6-persistence/src/main/java/com/netflix/conductor/es6/dao/index/ElasticSearchRestDAOV6.java

```diff
@@ -1117,6 +1117,11 @@ public long getWorkflowCount(String query, String freeText) {
         }
     }
 
+    @Override
+    public SearchResult<WorkflowSummary> getSummaries(SearchResult<String> searchResultIds) {
+        return null;
```

Comment:
I would change return value from null to:         

throw new UnsupportedOperationException("Not implemented yet.");

Reply:
Resolved.